### PR TITLE
CATROID-1371 REFACTOR SetBrightnessAction to Kotlin

### PR DESCRIPTION
**Pull request by NoQQ**
https://jira.catrob.at/browse/CATROID-1371

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/test/java/org/catrobat/catroid/test/content/actions/SetBrightnessActionTest.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
```
Comment (Walcho1125) [inline catroid/src/test/java/org/catrobat/catroid/test/content/actions/SetBrightnessActionTest.kt:3]:
```suggestion
 * Copyright (C) 2010-2022 The Catrobat Team
```

**Review by Walcho1125 — APPROVED** [review]
Looks good to me

**Review by adnaribo — APPROVED** [review]

**Comment by juliajulie95** [comment]
Please rebase :)

**Comment by NoQQ** [comment]
Now rebased

**Review by Walcho1125 — APPROVED** [review]
Thank you again!

**Pull request merged**